### PR TITLE
Set reviewer

### PR DESCRIPTION
**Pull request by tehreemtalat7**

### CODE REVIEW

File: app/controllers/tasks_controller.rb [hunk 1]
```diff
@@ -1,53 +1,69 @@
...
   end
 
   def update
     if !params[:assignable_employee_id].empty? && !params[:assignable_team_id].empty?
-      render :edit, notice: "Make sure you delete employee field or team field"
+      render :edit, notice: 'Make sure you delete employee field or team field'
     else
-      params[:task][:status] = "new" if params[:task][:assignable_id].empty?
-      params[:task][:status] = "assigned" if !params[:task][:assignable_id].empty?
+      params[:task][:status] = 'new' if params[:task][:assignable_id].empty?
```
Comment (mabdullahkhalil) [inline app/controllers/tasks_controller.rb]: don't hard code strings

File: app/controllers/tasks_controller.rb [hunk 2]
```diff
@@ -1,53 +1,69 @@
...
+      params[:task][:status] = 'assigned' if !params[:task][:assignable_id].empty?
```
Comment (mabdullahkhalil) [inline app/controllers/tasks_controller.rb]: Don't hard code strings

File: app/controllers/employees_controller.rb [hunk 3]
```diff
@@ -7,12 +8,8 @@ def employees_lists
     end
   end
 
-  def new
-    @employee = current_employee.company.employees.new
-  end
-
   def show
-    @attendances_list = current_employee.get_attendances_admin
+    @attendances_list = current_employee.company.attendances.where(status: Attendance::STATUS[:PRESENT]).order(login_time: :desc)
```
Comment (mabdullahkhalil) [inline app/controllers/employees_controller.rb]: Would suggest you to write this function in model

File: app/views/attendances/_attendance_table.html.erb [hunk 4]
```diff
@@ -28,7 +28,7 @@
         <%= attendance.login_time.to_formatted_s(:long_ordinal) %>
       </td>
       <td>
-        <% if attendance.logout_time.blank? %>
+        <% if logout_added?(attendance) %>
```
Comment (mabdullahkhalil) [inline app/views/attendances/_attendance_table.html.erb]: add all of this if condition to the helper function

File: app/views/projects/index.html.erb [hunk 5]
```diff
@@ -2,7 +2,7 @@
 <div id ="notice" class="alert alert-primary <%= 'd-none' if notice.blank? %>" role="alert">
   <%= notice %>
 </div>
-
+<%= link_to 'New Project', new_project_path, class: "btn btn-regular btn-primary" %>
 <% if !@projects.nil?%>
```
Comment (mabdullahkhalil) [inline app/views/projects/index.html.erb]: if @projects.any?

File: app/views/tasks/_form.html.erb [hunk 6]
```diff
@@ -4,41 +4,61 @@
 
 <%= form_for [@project, @task] do |form| %>
 
-<%= render partial: 'hidden', locals: { form: form } %>
+  <%= render partial: 'hidden', locals: { form: form } %>
```
Comment (mabdullahkhalil) [inline app/views/tasks/_form.html.erb]: extra lines

File: app/views/tasks/employee_tasks.html.erb [hunk 7]
```diff
@@ -0,0 +1,65 @@
+<% content_for :head do %>
+  <%= javascript_include_tag 'employee_tasks' %>
+<% end %>
+
```
Comment (mabdullahkhalil) [inline app/views/tasks/employee_tasks.html.erb]: extra lines

File: app/views/tasks/employee_tasks.html.erb [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+<h1>My Tasks</h1>
+
```
Comment (mabdullahkhalil) [inline app/views/tasks/employee_tasks.html.erb]: extra lines

File: app/views/tasks/employee_tasks.html.erb [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+<div class="alert alert-success alert-dismissable present" id="logtime_success_alert" style="display: none;">
+  <button type="button" class="close" data-dismiss="alert" aria-hidden="true">
+    ×
```
Comment (mabdullahkhalil) [inline app/views/tasks/employee_tasks.html.erb]: ??

File: app/views/tasks/employee_tasks.html.erb [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+  <table id="tasks_table" class="table table-striped table-bordered" style="width:100%">
+  <thead>
+    <tr>
+      <th>Task Name</th>
+      <th>Status</th>
+      <th>Expected End Date</th>
+      <th>Hours Worked</th>
+    </tr>
+  </thead>
+
```
Comment (mabdullahkhalil) [inline app/views/tasks/employee_tasks.html.erb]: extra lines

File: app/controllers/tasks_controller.rb (hunk 1 above)
Comment (tehreemtalat7) [reply app/controllers/tasks_controller.rb]: this is abdullah's code and he has refactored it in his brnach

File: app/views/attendances/_attendance_table.html.erb (hunk 4 above)
Comment (tehreemtalat7) [reply app/views/attendances/_attendance_table.html.erb]: it is in the helper

**Pull request merged**